app/keepers/keepers.go

```diff
@@ -432,6 +444,7 @@ func NewAppKeeper(
 	// - core IBC
 	// - ratelimit
 	// - pfm
+	// - ibcfee
 	// - transfer
```

Comment:
The comment doesn't reflect the order below. 
```suggestion
	// - core IBC
	// - ibcfee
	// - ratelimit
	// - pfm
	// - transfer
```

Reply:
Changed